internal/binutils/addr2liner.go

```diff
@@ -116,11 +120,7 @@ func newAddr2Liner(cmd, file string, base uint64) (*addr2Liner, error) {
 }
 
 func (d *addr2Liner) readString() (string, error) {
```

Comment:
Is this function needed now at all?

Reply:
I think it could be removed, but the rest of the code would use `d.rw.readLine` in that case. I don't have enough context to know whether that's the desired code or not.

Reply:
Created https://github.com/google/pprof/pull/596 to address the comments.